drivers/gles3/shaders/scene.glsl

```diff
@@ -1312,8 +1318,12 @@ void main() {
 
 	// Tonemap before writing as we are writing to an sRGB framebuffer
 	frag_color.rgb *= exposure;
-	frag_color.rgb = apply_tonemapping(frag_color.rgb, white);
-	frag_color.rgb = linear_to_srgb(frag_color.rgb);
+	if (scene_data.use_tonemap) {
+		frag_color.rgb = apply_tonemapping(frag_color.rgb, white);
+	}
+	if (scene_data.use_linearize) {
+		frag_color.rgb = linear_to_srgb(frag_color.rgb);
+	}
```

Comment:
The Gl Compatibility renderer renders in sRGB space, the linear->srgb conversion should never be disabled